tensorflow/compiler/tf2xla/cpu_function_runtime.h

```diff
@@ -104,6 +104,7 @@ class BufferInfo {
  private:
   BufferInfo() = default;
 
+  // Enum type needs to match size of member "size_" so bitfields get merged.
```

Comment:
Please also mention that this happens only on MSVC.  Otherwise someone will try this on clang, see that it works and remove the workaround.

Reply:
Also, do you mind adding a `static_assert` that would catch this?

Reply:
Actually, this applies to Clang on Windows running in MSVC or Mingw mode. Just tested it myself.